### PR TITLE
Changes to support agent keys and dockerhub deployment

### PR DESCRIPTION
**Pull request by mikecohen**
This change is [Reviewable](https://reviewable.io/reviews/flowmill/flowmill-k8s/8)

### CODE REVIEW

File: values.yaml [hunk 1]
```diff
@@ -3,19 +3,43 @@
...
 agent:
-  repository: 966881513036.dkr.ecr.us-east-1.amazonaws.com/flowmill/flowmill-agent
-  tag: 0.7.15.c97d8410
+  repository: docker.io/flowmill/agent
+  tag: v0.8.0
+  pullPolicy: IfNotPresent
+  resources: {}
+  args:
+    - --config-file
+    - /etc/flowmill/config.yaml
```
Comment (cirego) [inline values.yaml]: This should probably be moved back into the `template`.

File: values.yaml [hunk 2]
```diff
@@ -3,19 +3,43 @@
...
 relay:
-  repository: 966881513036.dkr.ecr.us-east-1.amazonaws.com/flowmill/flowmill-k8s-relay
-  tag: v0.7.14-139-g3b1e2590
+  repository: docker.io/flowmill/k8s-relay
+  tag: v0.8.0
   pullPolicy: IfNotPresent
   resources: {}
+  args:
+    - --config-file
+    - /etc/flowmill/config.yaml
```
Comment (cirego) [inline values.yaml]: Likewise from above, this should probably be moved back into the `template` file.

File: values.yaml (hunk 1 above)
Comment (juchem) [reply values.yaml]: I'm not sure it should. This file is optional and contains user defined labels.
We probably need better documentation for it. Not sure who's using it.

Comment (cirego) [reply values.yaml]: What if we expose those settings as values that the user can set. That would then trigger creation / addition of `config.yaml` based on those values being set?

**Pull request merged**